src/rgw/driver/rados/rgw_sal_rados.h

```diff
@@ -19,7 +19,7 @@
 #include "rgw_rados.h"
 #include "rgw_notify.h"
 #include "rgw_oidc_provider.h"
-#include "rgw_role.h"
+#include "../../rgw_role.h"
```

Comment:
can we rename `driver/rados/rgw_role.h` to `driver/rados/rgw_metadata_role.h` to avoid this conflict?

in general, i don't think we should use the `rgw_` filename prefix for sources in subdirectories of src/rgw/, but we can talk about that later

Reply:
will be fixed in the next push. I.e. rename driver/rados/rgw_role.h -> driver/rados/rgw_metadata_role.h    Mass renaming I'd prefer to do in a separate PR